www/docs/en/6.x/config_ref/images.md

```diff
@@ -41,6 +41,7 @@ platform      | *Optional* <br/> Target platform
 width         | *Optional* <br/> Icon width in pixels
 height        | *Optional* <br/> Icon height in pixels
 density       | *Optional* <br/> Specified icon density (Android Specific)
+target        | *Optional* <br/> Destination filename for the image file and all its' MRT companions (Windows Specific)
```

Comment:
For platform specific, content - we use ==Windows==<br/> as it shows up as a <mark/> element and is custom formatted on the website.